target_parquet/helpers.py

```diff
@@ -45,21 +45,16 @@ def flatten(dictionary, flat_schema, parent_key="", sep="__"):
              'key_7__key_9': None
          }
     """
-    items = []
+    items = {} if parent_key else flat_schema.fromkeys(flat_schema.keys(), None)
```

Comment:
Since this method is called recursively, I've included the empty record only in the first call